matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/network/ProgressRequestBody.kt

```diff
@@ -39,15 +40,9 @@ internal class ProgressRequestBody(private val delegate: RequestBody,
 
     override fun isDuplex() = delegate.isDuplex()
 
-    override fun contentLength(): Long {
-        try {
-            return delegate.contentLength()
-        } catch (e: IOException) {
-            e.printStackTrace()
-        }
+    val length = tryThis { delegate.contentLength() } ?: -1
```

Comment:
So this is computed only once, is it OK?

Reply:
yes